optimum/intel/neural_compressor/quantization.py

```diff
@@ -179,6 +181,9 @@ def quantize(
                 data_collator=data_collator,
             )
 
+        if isinstance(self._original_model.config, PretrainedConfig):
+            self._original_model.config.backend = quantization_config.backend
+
```

Comment:
Also could we add a test for post-training quantization with IPEX ?

Reply:
yes.

Reply:
Thanks !